staging/src/k8s.io/code-generator/cmd/validation-gen/output_tests/inner/validatefalse/doc.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +k8s:validation-gen=TypeMeta
+// +k8s:validation-gen-scheme-registry=k8s.io/code-generator/cmd/validation-gen/testscheme.Scheme
+
+// Package inner contains test types for testing inner field validation tags.
+package inner
+
+import "k8s.io/code-generator/cmd/validation-gen/testscheme"
+
+var localSchemeBuilder = testscheme.New()
+
+// M1 represents the formerly anonymous struct used in T1.Simple.
```

Comment:
This comment doesn't make any sense - copied from a different one?

Reply:
Yes apologies, thanks for catching this. Updated now to :

// T2 has string, slice, pointer and map fields to test inner field validations across types